.github/workflows/buf-lint.yml

```diff
@@ -1,5 +1,5 @@
 name: buf.build
-on:
+on:  # yamllint disable-line rule:truthy
```

Comment:
Hi @roidelapluie ,
I created #11871 so you dont have to add this comment on each github workflow file

Reply:
thanks, however this branch is just mirroring main so it's not relevant for this specific pr.